cloe_shell.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+export CLOE_LOG_LEVEL=debug
+export CLOE_STRICT_MODE=1
+export CLOE_ROOT="$(pwd)"
```

Comment:
hmm is pwd correct here or does it really need to be the root of this repository? If yes maybe consider using basename of this script shenanigans

Reply:
It is correct here because cloe_shell is started by BUILD.bazel target which has the root dir. It's just defined for compatibility with some of the tests that want to know.